README.md

```diff
@@ -1,63 +1,3280 @@
+Jan 13 11:05:07 dee2945 systemd[1]: Reached target Network.
+Jan 13 11:05:07 dee2945 systemd[1]: Started OpenVPN.
+Jan 13 11:05:07 dee2945 systemd[1]: Starting Resin init service...
+Jan 13 11:05:07 dee2945 systemd[1]: Starting DNS forwarder and DHCP server...
+Jan 13 11:05:07 dee2945 systemd[1]: Started OS configuration update service.
+Jan 13 11:05:07 dee2945 NetworkManager[740]: <info>  [1578913507.3047] manager[0x12ec000]: monitoring kernel firmware directory '/lib/firmware'.
+Jan 13 11:05:07 dee2945 bash[758]: Board specific initialization...
+Jan 13 11:05:07 dee2945 dnsmasq[759]: dnsmasq: syntax check OK.
+Jan 13 11:05:07 dee2945 systemd[1]: Started DNS forwarder and DHCP server.
+Jan 13 11:05:07 dee2945 systemd[1]: Starting Balena Application Container Engine...
+Jan 13 11:05:07 dee2945 systemd[1]: Starting Resin proxy configuration service...
+Jan 13 11:05:07 dee2945 dnsmasq[763]: dnsmasq[763]: started, version 2.78 cachesize 150
+Jan 13 11:05:07 dee2945 dnsmasq[763]: dnsmasq[763]: compile time options: IPv6 GNU-getopt DBus no-i18n no-IDN DHCP DHCPv6 no-Lua TFTP no-conntrack ipset auth no-DNSSEC >
+Jan 13 11:05:07 dee2945 dnsmasq[763]: dnsmasq[763]: DBus support enabled: connected to system bus
+Jan 13 11:05:07 dee2945 dnsmasq[763]: dnsmasq[763]: reading /etc/resolv.dnsmasq
+Jan 13 11:05:07 dee2945 dnsmasq[763]: dnsmasq[763]: using nameserver 8.8.8.8#53
+Jan 13 11:05:07 dee2945 dnsmasq[763]: dnsmasq[763]: read /etc/hosts - 6 addresses
+Jan 13 11:05:07 dee2945 dnsmasq[763]: dnsmasq[763]: using nameserver 8.8.8.8#53
+Jan 13 11:05:07 dee2945 kernel: i2c /dev entries driver
+Jan 13 11:05:07 dee2945 kernel[664]: [   14.974497] i2c /dev entries driver
+...
+```
+The rest of the journal has been truncated for brevity. Note that the `-a`
+(short for `--all`) has been used, so that the logs from the service containers
+themselves are also visible. Without this switch, the service container output
+from `balena.service` (the balenaEngine) would be output like this:
+```
+Jan 13 11:05:20 dee2945 69dc1cee03e4[765]: [1B blob data]
+Jan 13 11:05:20 dee2945 69dc1cee03e4[765]: [36B blob data]
+Jan 13 11:05:20 dee2945 69dc1cee03e4[765]: [16B blob data]
+Jan 13 11:05:20 dee2945 69dc1cee03e4[765]: [1B blob data]
+Jan 13 11:05:20 dee2945 e374bbb9ddd4[765]: [1B blob data]
+Jan 13 11:05:20 dee2945 e374bbb9ddd4[765]: [35B blob data]
+Jan 13 11:05:20 dee2945 e374bbb9ddd4[765]: [16B blob data]
+Jan 13 11:05:20 dee2945 e374bbb9ddd4[765]: [1B blob data]
+```
+This obviously is not a problem if you're not looking for issues in the
+application's services, but usually enabling persistent logs allows us to
+examine what also might be happening in a service container that could cause
+an issue (for example, maybe the container is privileged and causing a reboot
+because it talks to the host DBus and issues a restart).
+
+The total space for persistent journals is currently 8MB (the total for the
```

Comment:
In OS 2.46 it has been bumped up to 32MB and moved data partition I believe.

Reply:
I just happened to be updating the docs for this and it was 2.45 (and yes to state->data partition) per https://github.com/balena-os/meta-balena/blob/development/CHANGELOG.md#v2450

Reply:
Thanks both. Done. :)